### PR TITLE
login and registration "sad path" cypress tests

### LINKED ISSUE #431: Create "sad path" tests for existing login and registration Cypress tests
# Feature request
We only test "happy path" for logging in and registering. I'd like to also maintain e2e coverage for the cases when the user isn't successful in doing either.

**Describe the solution you'd like**
**Login**
- Attempts to login with information that passes form validation, but is _not_ an actual user.
- Attempts to login with an invalid email
- Attempts to login with an invalid password
- Attempts to login and the API response is coerced to fail (i.e. our server is down)

**Registration**
- Attempts to register using an email that already exists
- Attempts to register using an invalid email
- Attempts to register using an invalid email match
- Attempts to register using an invalid password
- Attempts to register using an invalid password match
- Attempts to register using an invalid zipcode
- Attempts to register without filling out all required fields

### PR DESCRIPTION
**Pull request by KrDimitrov**
# Description of changes

Implemented new cypress tests for login and registration forms, where a user is expected to mess something up.
# Issue Resolved

Fixes #431

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit 15d21bb0882482be0cfbbcb3cc936176d15be624

https://deploy-preview-440--operation-code-storybook.netlify.com

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit e402b14203d87648f2a2b85617146fd1bc72338b

https://deploy-preview-440--operation-code-storybook.netlify.com

File: cypress/integration/register.spec.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ describe('register', function() {
   const newUser = {
     email: faker.internet.email(),
 
+    existingEmail: 'kylemh.email12@gmail.com',
```
Comment (kylemh) [inline cypress/integration/register.spec.js]: This doesn't really belong as part of `newUser` could you declare it as it's own variable?

`const existingEmail = 'kylemh.email12@gmail.com';`

**Comment by kylemh** [comment]
Absolutely amazing work here. I'll resolve the error handling at some point tonight, and merge your PR

**Comment by kylemh** [comment]
Oh shoot. I know the mock had the button's as primary, but against the white card background it's an accessibility violation. Let's just make them secondary buttons

**Comment by kylemh** [comment]
NOTE: The build failure is expected. See conversation in #431 to understand more. I will have a PR by EOD to resolve the issue.

File: cypress/integration/register.spec.js (hunk 1 above)
Comment (KrDimitrov) [reply cypress/integration/register.spec.js]: Good catch! Im not sure why i declared it there. Fixed!

**Pull request merged**